src/Form/DataTransformer/BooleanToStringTransformer.php

```diff
@@ -33,11 +33,17 @@ public function __construct(string $trueValue)
         $this->trueValue = $trueValue;
     }
 
+    /**
+     * @param bool $value
+     */
     public function transform($value): ?string
     {
         return $value ? $this->trueValue : null;
     }
 
+    /**
+     * @param mixed $value
```

Comment:
```suggestion
     * @param string|null $value
```
I guess

Reply:
I let mixed because `filter_var` works with `mixed` so it could work with anything.